tests/bson/bson-javascript-set_state-001.phpt

```diff
@@ -31,39 +31,39 @@ echo "\n\n";
 <?php exit(0); ?>
 --EXPECTF--
 %r\\?%rMongoDB\BSON\Javascript::__set_state(array(
-%w'code' => 'function foo(bar) { return bar; }',
-%w'scope' => NULL,
+   'code' => 'function foo(bar) { return bar; }',
+   'scope' => NULL,
 ))
 
 %r\\?%rMongoDB\BSON\Javascript::__set_state(array(
-%w'code' => 'function foo(bar) { return bar; }',
-%w'scope' => 
-  %Sarray(
-  %S),
+   'code' => 'function foo(bar) { return bar; }',
+   'scope' =>%w
+  %r\(object\)? %rarray(
+  %r\)?%r),
 ))
 
 %r\\?%rMongoDB\BSON\Javascript::__set_state(array(
-%w'code' => 'function foo() { return foo; }',
-%w'scope' => 
+   'code' => 'function foo() { return foo; }',
+   'scope' =>%w
   %Sarray(
-  %w'foo' => 42,
+     'foo' => 42,
   %S),
 ))
 
 %r\\?%rMongoDB\BSON\Javascript::__set_state(array(
-%w'code' => 'function foo() { return id; }',
-%w'scope' => 
+   'code' => 'function foo() { return id; }',
+   'scope' =>%w
   %Sarray(
-  %w'id' => 
+     'id' =>%w
```

Comment:
Was `%w` actually necessary here (and above) _after_ the `=>` for portability across PHP versions?

Reply:
`var_export` has some strange formatting when a property contains an object or array: in that case, one space of the indentation is actually printed at the end of the previous line. As mentioned in my comment above, I've replaced those with `%w` to avoid editors from automatically trimming whitespace at the end and to make it more visible. I found some other occurrences, so those have been adjusted as well.